plugins/SliceInfoPlugin/MoreInfoWindow.qml

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// PluginBrowser is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.7
+import QtQuick.Window 2.2
+import QtQuick.Controls 1.4
+import QtQuick.Controls.Styles 1.4
+
+import UM 1.3 as UM
+import Cura 1.0 as Cura
+
+
+UM.Dialog
+{
+    id: baseDialog
+    title: catalog.i18nc("@title:window", "More information on anonymous data collection")
+    visible: false
+
+    minimumWidth: 500 * screenScaleFactor
+    minimumHeight: 400 * screenScaleFactor
+    width: minimumWidth
+    height: minimumHeight
+
+    property bool allowSendData: true  // for saving the user's choice
+
+    onAccepted: manager.setSendSliceInfo(allowSendData)
+
+    onVisibilityChanged:
+    {
+        if (visible)
+        {
+            baseDialog.allowSendData = UM.Preferences.getValue("info/send_slice_info");
+            if (baseDialog.allowSendData)
+            {
+                allowSendButton.checked = true;
+            }
+            else
+            {
+                dontSendButton.checked = true;
+            }
+        }
+    }
+
+    Item
+    {
+        Connections
+        {
+            target: CuraApplication.getCuraAppSignals()
+            onShowMoreInfoOnAnonymousDataCollection:
+            {
+                baseDialog.show();
+            }
+        }
+
+        id: textRow
+        anchors
+        {
+            top: parent.top
+            bottom: radioButtonsRow.top
+            bottomMargin: UM.Theme.getSize("default_margin").height
+            left: parent.left
+            right: parent.right
+        }
+
+        Label
+        {
+            id: headerText
+            anchors
+            {
+                top: parent.top
+                left: parent.left
+                right: parent.right
+            }
+
+            text: catalog.i18nc("@text:window", "Cura sends anonymous data so we can improve the print quality and user experience. Below is an example of all the data we send.")
```

Comment:
I don't think the usage of the word "we" is appropriate here. It makes sense when talking as developers to users but here they are just interacting with the application. How about:

    Cura sends anonymous data to Ultimaker in order to improve the print quality and user experience. Below is an example of all the data that is sent.

Reply:
Changed in 4d3acddfa4b434165c52e99f2d0d9969b614edbe